scheduler-driver/src/main/java/gov/nasa/jpl/aerie/scheduler/solver/PrioritySolver.java

```diff
@@ -290,86 +233,9 @@ public void initializePlan() throws SimulationFacade.SimulationException, Schedu
     checkAndInsertActs(problem.getInitialPlan().getActivitiesByTime(), true);
```

Comment:
Flagging it on a usage, but the boolean parameter is unused. I'm unclear if this was unused in the PR it was introduced in or if changes in this PR has made it unused.

Reply:
it was made obsolete by this pr